src/interface/swagger.json

```diff
@@ -939,6 +939,24 @@
                     }
                 }
             }
+        },
+        "/health": {
+        "get": {
+            "description": "The HTTP request GET /health is used to return the current status of the API.",
+            "summary": "Get Server",
+            "tags": [
+                "health"
+                ],
+            "operationId": "HealthGet",
+            "produces": [
+                "application/json"
+                ],
+            "responses": {
+                "200": {
+                    "$ref": "#/responses/ResponseHealth200"
+                    }
+                }
```

Comment:
where are the error responses? you would need these. if there is a failure the swagger would know how to response and what is valid

Reply:
Adding the Error response code to the swagger.